src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -349,17 +349,147 @@ <h3 *ngIf="displaySubsection(showQuestion, 4, 7)">Managing Session Responses</h3
+                <div ngbTooltip="Group results in the current view by team">
+                  <input type="checkbox"> Group by Teams
+                </div>
+                <div ngbTooltip="Show statistics">
+                  <input type="checkbox"> Show Statistics
+                </div>
+                <div ngbTooltip="Indicate missing responses">
+                  <input type="checkbox"> Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </tm-example-box>
           <p>
             In the example below, results are sorted by <b>Giver > Recipient > Question</b>. Additionally, missing responses
             have been recorded.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-instructor-session-result-grq-view [session]="this.exampleFeedbackSession"
+                                                   [section]="'Section A'"
+                                                   [sectionType]="InstructorSessionResultSectionType.EITHER"
+                                                   [groupByTeam]="true"
+                                                   [showStatistics]="true"
+                                                   [indicateMissingResponses]="true"
+                                                   [isExpandAll]="true"
+                                                   [instructorCommentTableModel]="exampleInstructorCommentTableModel"
+                                                   [responses]="exampleGrqResponses"
+            ></tm-instructor-session-result-grq-view>
+            <div class="card-header bg-info text-dark course-tab-header">
+              Participants who have not responded to any question
+              <div class="float-right">
+                <i class="fas fa-chevron-up"></i>
+              </div>
+              <div class="float-right" style='padding-right:15px; margin:-5px;'>
+                <button class="btn btn-light btn-sm button" type="button">
+                  Remind All
+                </button>
+              </div>
+            </div>
+            <div class="card-body">
+              <div>
+                <table class="table">
+                  <thead>
+                  <tr>
+                    <th>Team</th>
+                    <th>Name</th>
+                    <th>Actions</th>
+                  </tr>
+                  </thead>
+                  <tbody>
+                  <tr *ngFor="let noResponseStudent of exampleStudents">
+                    <td>{{ noResponseStudent.teamName }}</td>
+                    <td>{{ noResponseStudent.name }}</td>
+                    <td>
+                      <a class="btn btn-light btn-sm button disabled">
+                        Submit response
+                      </a>
+                    </td>
+                  </tr>
+                  </tbody>
+                </table>
+              </div>
+            </div>
```

Comment:
I think a better way would be to reuse the no-response-panel template, pass in a new @Input displayOnly to control the `disable` of the buttons.

Reply:
If we want the button to still be clickable, the click function can then take in the `isDisplayOnly` attribute, do if-else check to decide whether to perform original task or nothing. 

Similar implementation can be found in `question-edit-form.component`.

Reply:
Awaiting opinions from the senior devs on this approach @wkurniawan07  @xpdavid

Reply:
> can then take in the isDisplayOnly attribute

You don't need to take in the value. It can be access by `this. isDisplayOnly ` if you define it as `input` property.

Reply:
Got it. Will do the change.